app/models/invoice.rb

```diff
@@ -4,5 +4,5 @@ class Invoice < ApplicationRecord
   has_many :items, through: :invoice_items
   belongs_to :customer
   has_many :transactions
-  enum :status, [:cancelled, "In Progress", :completed]
+  enum :status, ["Cancelled", "In Progress", "Completed"]
```

Comment:
Good catch here - this was creating errors with rails db:{drop,create,migrate,seed} earlier but I didn't catch it. hoping that having them as strings doesn't create any problems down the road. 🤞🏻